bundles/org.openhab.binding.xmltv/README.md

```diff
@@ -0,0 +1,87 @@
+| filePath                | X        | Full path (including filename) to an Xml TV file    |         |
+| refresh                 | X        | XMLTV file reload interval in hours                 | 24h     |
+
+## Thing Configuration
+
+| Configuration Parameter | Required | Description                                                     | Default |
+|-------------------------|----------|-----------------------------------------------------------------|---------|
+| channelId               | X        | Id of the channel as presented in the XmlTV file.               |         |
+| offset                  | X        | Offset applied to program times (forward or backward (minutes)  | 0       |
+| refresh                 | X        | refresh interval in seconds                                     | 60      |
+
+## Channels
+
+| Channel Type ID | Item Type            | Description                         |
+|-----------------|----------------------|-------------------------------------|
+| iconUrl         | String               | Channel Icon URL                    |
+| icon            | Image                | Icon of the channel                 |
+
+### Current program (currentprog) Channels Group
+
+| Channel Type ID | Item Type            | Description                                 |
+|-----------------|----------------------|---------------------------------------------|
+| progStart       | DateTime             | Program Start Time                          |
+| progEnd         | DateTime             | Program End Time                            |
+| progTitle       | String               | Program Title.                              |
+| progCategory    | String               | Program Category.                           |
+| progIconUrl     | String               | URL to an image of the program.             |
+| icon            | Image                | Icon of the program                         |
+| elapsedTime     | Number:Time          | Current time of currently playing program   |
+| remainingTime   | Number:Time          | Time remaining until end of the program     |
+| progress        | Number:Dimensionless | Relative progression of the current program |
+
+### Next program (nextprog) Channels Group
+
+| Channel Type ID | Item Type            | Description                                 |
+|-----------------|----------------------|---------------------------------------------|
+| progStart       | DateTime             | Program Start Time                          |
+| timeLeft        | Number:Time          | Time left before program start.             |
+| progEnd         | DateTime             | Program End Time                            |
+| progTitle       | String               | Program Title.                              |
+| progCategory    | String               | Program Category.                           |
+| progIconUrl     | String               | URL to an image of the program.             |
```

Comment:
Some of the descriptions now end with a period and some don't. I'd just remove them all (also from the other tables) so it's consistent.

```suggestion
| progIconUrl     | String               | URL to an image of the program              |
```